nnstreamer/tensor_filter/tensor_filter_nntrainer.cc

```diff
@@ -210,8 +218,22 @@ static int nntrainer_setInputDim(const GstTensorFilterProperties *prop,
     static_cast<NNTrainerInference *>(*private_data);
   g_return_val_if_fail(prop && nntrainer && in_info && out_info, -EINVAL);
 
-  auto num_input = in_info->num_tensors;
-  g_return_val_if_fail(num_input != 0, -EINVAL);
+  auto model_inputs = nntrainer->getInputDimension();
```

Comment:
Good point. when the setInputDim is called, we need to set the dim from nntrainer information.